.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
```

Comment:
I kind of feel like this intro is a little too much, couldn't this all go in the contribution guidelines and then just be a link with a pointer to the heading of this info?